kubernetes/helm/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-name: ollama-webui
-description: "Ollama Web UI: A User-Friendly Web Interface for Chat Interactions 👋"
+name: open-webui
+description: "open-webui UI: A User-Friendly Web Interface for Chat Interactions 👋"
```

Comment:
is it open-webui or Open webUI or ?

Reply:
> is it open-webui or Open webUI or ?

Open WebUI, Open-WebUI, open-webui in order of preference/as appropriate for application.